libs/pages/logs/infra/src/lib/page-infra-logs.tsx

```diff
@@ -30,24 +31,7 @@ export function PageInfraLogs() {
       : undefined
 
   return (
-    <LayoutLogs
-      data={cluster?.logs}
-      tabInformation={
-        cluster && (
-          <CardCluster
-            id={cluster?.id}
-            version={cluster?.version}
-            status={cluster?.status}
-            name={cluster?.name}
-            // last_execution_id={cluster?.last_execution_id}
-            last_execution_id={'ffffffff-ffffffff'}
-            cloud_provider={cluster?.cloud_provider}
-            region={cluster?.region}
-            organizationId={organizationId}
-          />
-        )
-      }
-    >
+    <LayoutLogs data={cluster?.logs} tabInformation={<CardClusterFeature />}>
```

Comment:
Here, in the end, LayoutLogs will rerender all the time CardClusterFeature is updated because it's a prop. So in the end moving everything in a feature did not change too much things